app/services/idv/session.rb

```diff
@@ -95,7 +95,7 @@ def create_profile_from_applicant_with_password(user_password)
     end
 
     def opt_in_param
-      opted_in_to_in_person_proofing unless !IdentityConfig.store.in_person_proofing_opt_in_enabled
+      opted_in_to_in_person_proofing if Identity::Hostdata.config.in_person_proofing_opt_in_enabled
```

Comment:
Had to change this because of line length

`unless` is already negated, so I think this actually reads clearer as an `if` (and bonus that `if` is fewer characters)